docs/content/en/docs/tutorials/tetragon-metrics.md

```diff
@@ -0,0 +1,112 @@
+kube-dns   ClusterIP   10.96.0.10     <none>        53/UDP,53/TCP,9153/TCP   3m
+tetragon   ClusterIP   10.96.54.218   <none>        2112/TCP                 3m
+```
+
+Describe the tetragon service to see the port on which metrics are being exposed.
+
+```bash
+➜  ~ kubectl describe service -n kube-system tetragon
+Name:              tetragon
+Namespace:         kube-system
+Labels:            app.kubernetes.io/instance=tetragon
+                   app.kubernetes.io/managed-by=Helm
+                   app.kubernetes.io/name=tetragon
+                   helm.sh/chart=tetragon-0.11.0
+Annotations:       meta.helm.sh/release-name: tetragon
+                   meta.helm.sh/release-namespace: kube-system
+Selector:          app.kubernetes.io/instance=tetragon,app.kubernetes.io/managed-by=Helm,app.kubernetes.io/name=tetragon,helm.sh/chart=tetragon-0.11.0
+Type:              ClusterIP
+IP Family Policy:  SingleStack
+IP Families:       IPv4
+IP:                10.96.54.218
+IPs:               10.96.54.218
+# Metrics Port is 2112
+Port:              metrics  2112/TCP
+TargetPort:        2112/TCP
+Endpoints:         172.18.0.2:2112
+Session Affinity:  None
+Events:            <none>
+```
+
+### Port Forwarding
+
+Use `kubectl port forward` to expose the metrics.
+
+```bash
+➜  ~ kubectl -n kube-system port-forward service/tetragon 2112:2112
+Forwarding from 127.0.0.1:2112 -> 2112
+Forwarding from [::1]:2112 -> 2112
+```
+
+### Fetch the metrics
+Then the exposed metrics can be fetched using `curl` command at `localhost:2112/metrics`
+
+```shell
+➜  ~ curl localhost:2112/metrics
+
+# HELP promhttp_metric_handler_errors_total Total number of internal errors encountered by the promhttp metric handler.
+# TYPE promhttp_metric_handler_errors_total counter
+promhttp_metric_handler_errors_total{cause="encoding"} 0
+promhttp_metric_handler_errors_total{cause="gathering"} 0
+# HELP tetragon_errors_total The total number of Tetragon errors. For internal use only.
+# TYPE tetragon_errors_total counter
+...
+```
+
+## Container
+
+Tetragon, when installed as a container as mentioned in [Container Deployment]({{<ref "docs/getting-started/deployment/container" >}})
+
+## Package
+
+Tetragon, when installed via release packages as mentioned in [Package Deployment](../getting-started/deployment/package.md). It can be interacted with `tetragon` command. By default, metrics are disabled, which can be enabled using `--metrics-server` flag, by specifying the address.
+
+### Set Metrics Address
+
+Run `sudo tetragon --metrics-server localhost:2112` to set metrics address to `localhost:2112` and export metrics.
+
+```bash
+➜  ~ sudo tetragon --metrics-server localhost:2112
+[sudo] password for <user>: 
+time="2023-09-21T13:17:08+05:30" level=info msg="Starting tetragon" version=v0.11.0
+time="2023-09-21T13:17:08+05:30" level=info msg="config settings" config="mapeased  
+...
+...
+time="2023-09-21T13:17:08+05:30" level=info msg="Listening for events..."
+```
```

Comment:
````suggestion
```shell-session
sudo tetragon --metrics-server localhost:2112
```

The output should be similar to this:

```
time="2023-09-21T13:17:08+05:30" level=info msg="Starting tetragon" version=v0.11.0
time="2023-09-21T13:17:08+05:30" level=info msg="config settings" config="mapeased  
[...]
time="2023-09-21T13:17:08+05:30" level=info msg="Listening for events..."
```
````
Can you put the log line which specifies the metrics port? There is one if I remember correctly.

Reply:
sure.